packages/shared/fastfetchbot_shared/utils/number.py

```
@@ -0,0 +1,8 @@
+def positive_int(value) -> int | None:
+    if value is None or isinstance(value, bool):
+        return None
+    try:
+        number = int(round(float(value)))
```

Ruff (0.15.14)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)